### PR TITLE
Fix the problem of write loss when the buffer is too large

### PR DESCRIPTION
**Pull request by 942011334**
# [BUG]/[FEATURE] title

### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

File: server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/storage/FlushTask.scala [hunk 1]
```diff
@@ -33,7 +33,12 @@ private[worker] class LocalFlushTask(
     fileChannel: FileChannel,
     notifier: FlushNotifier) extends FlushTask(buffer, notifier) {
   override def flush(): Unit = {
-    fileChannel.write(buffer.nioBuffers())
+    val buffers = buffer.nioBuffers()
+    for (buffer <- buffers) {
+      while(buffer.hasRemaining) {
```
Comment (waitinfuture) [inline server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/storage/FlushTask.scala:38]: Plz add a whitespace after while

**Comment by 942011334** [comment]
ok




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Kent ***@***.***&gt;; 
发送时间: 2022年9月1日(星期四) 下午3:20
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alibaba/RemoteShuffleService] Fix the problem of write loss when the buffer is too large (PR #495)





 
@waitinfuture commented on this pull request.
 
 
In server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/storage/FlushTask.scala:
 &gt; @@ -33,7 +33,12 @@ private[worker] class LocalFlushTask(      fileChannel: FileChannel,      notifier: FlushNotifier) extends FlushTask(buffer, notifier) {    override def flush(): Unit = { -    fileChannel.write(buffer.nioBuffers()) +    val buffers = buffer.nioBuffers() +    for (buffer <- buffers) { +      while(buffer.hasRemaining) {  
Plz add a whitespace after while

File: server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/storage/FlushTask.scala (hunk 1 above)
Comment (942011334) [reply server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/storage/FlushTask.scala]: done

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks!

**Review by FMX — APPROVED** [review]
LGTM.

**Review by AngersZhuuuu — COMMENTED** [review]
Good catch!

**Pull request merged**